src/explicitode.jl

```diff
@@ -103,12 +103,13 @@ function stepsize{T<:Real, U<:Number}(x::Taylor1{U}, epsilon::T)
     ord = x.order
     h = convert(T, Inf)
     for k in (ord-1, ord)
-        @inbounds aux = abs( x[k+1] )
+        @inbounds aux = norm( x[k+1], Inf)
         aux == zero(T) && continue
         aux = epsilon / aux
         kinv = one(T)/k
         aux = aux^kinv
         h = min(h, aux)
+        h
```

Comment:
This line should also be removed?

Reply:
done too! Actually I don't know how that line got there xD.

Reply:
Great!